# Index of the segment that is specified. Counting starts from "0"
uint8 segment_index

# List of specified parameters
diagnostic_msgs/KeyValue[] parameters